x-pack/plugins/security_solution/public/common/components/endpoint/agent_status_text.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { HostStatus } from '../../../../common/endpoint/types';
+
+export const getAgentStatusText = (hostStatus: HostStatus) => {
```

Comment:
Ok. I see what you did here... When I saw `getAgentStatusText()` I though you were using the Fleet helper function.

Since this is under `common/components/`, can you make it a component? as in:

```typescript
<EndpointAgentStatusText hostStatus="" />
```

Reply:
My original intention was just to have this generate the text for the status which can be used either in plain text reporting or in a containing component like it is here: https://github.com/elastic/kibana/pull/135935/files#diff-dcba59229f5aeb7b5e1cc30e8b37d4b848460e6e240da640eac479b392e28939R21

Thought - are you OK if I export this text from the above file so that we can either import the text or the entire `AgentStatus` component based on the need?

Reply:
yeah, that's fine - nothing that should hold this up. It just feels more like a "service" or "utility" than a component, so I was just highlighting that it probably should not be under `component/`